src/cpp/ext/otel/otel_plugin.cc

```diff
@@ -61,26 +61,24 @@ absl::string_view OpenTelemetryTargetKey() { return "grpc.target"; }
 namespace {
 absl::flat_hash_set<std::string> BaseMetrics() {
```

Comment:
Does this need to be `std::string` instead of `absl::string_view`?

Reply:
I want to intentionally store the set as `std::string` instead of `absl::string_view`. The case I'm worried about is when in the future we allow users to provide the set of metric names that they want enabled and they provide non-constant strings